### DISCUSSION TITLE
How to fetch data similar to store function

### DISCUSSION THREAD

**Discussion by srivatsan2607** [GrapesJS/grapesjs#4793] [Q&A]
Hi @artf ,I hope you are doing well. I am trying to store the editor contents on a button click apart from the storage command.
The data from storage:store command is in JSON format with all the styles and pages. Is there any other way to get the same on a button click?

**Comment by artf** [comment]
https://grapesjs.com/docs/api/editor.html#getprojectdata

**Reply by srivatsan2607** [reply]
Saved my day! Thank you.
